js/queries.js

```diff
@@ -51,14 +58,18 @@ function getSourceText(state, actor) {
 }
 
 function getBreakpoint(state, location) {
+  location = Immutable.fromJS(location);
   return getBreakpoints(state).get(makeLocationId(location));
 }
 
 /**
  * @param object - location
  */
 function makeLocationId(location) {
-  return location.actor + ":" + location.line.toString();
+  if(!Immutable.Iterable.isIterable(location)) {
+    location = Immutable.fromJS(location);
+  }
+  return location.get("actor") + ":" + location.get("line").toString();
```

Comment:
I see why you need to check the location type here and `getBreakpoint`, but it's disappointing.